security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,153 +12,28 @@ These release notes track the development of {cert-manager-operator}.
-
-[id="cert-manager-log-level-1.11.1"]
-==== Setting log levels for cert-manager and the {cert-manager-operator}
-* To troubleshoot issues with cert-manager and the {cert-manager-operator}, you can now configure the log level verbosity by setting a log level for cert-manager and the {cert-manager-operator}. For more information, see xref:../../security/cert_manager_operator/cert-manager-log-levels.adoc#cert-manager-log-levels[Configuring log levels for cert-manager and the {cert-manager-operator}].
-
-[id="cert-manager-authentication-aws-1.11.1"]
-==== Authenticating the {cert-manager-operator} with AWS
-* You can now configure cloud credentials for the {cert-manager-operator} on AWS clusters with Security Token Service (STS) and without STS. For more information, see xref:../../security/cert_manager_operator/cert-manager-authenticate-aws.adoc#cert-manager-authenticate-aws[Authenticating the {cert-manager-operator} on AWS  Security Token Service] and xref:../../security/cert_manager_operator/cert-manager-authentication-non-sts.adoc#cert-manager-authentication-non-sts[Authenticating the {cert-manager-operator} on AWS].
-
-[id="cert-manager-authentication-gcp-1.11.1"]
-==== Authenticating the {cert-manager-operator} with GCP
-* You can now configure cloud credentials for the {cert-manager-operator} on GCP clusters with Workload Identity and without Workload Identity. For more information, see xref:../../security/cert_manager_operator/cert-manager-authenticate-gcp.adoc#cert-manager-authenticate-gcp[Authenticating the {cert-manager-operator} with GCP Workload Identity] and xref:../../security/cert_manager_operator/cert-manager-authenticate-non-sts-gcp.adoc#cert-manager-authenticate-non-sts-gcp[Authenticating the {cert-manager-operator} with GCP]
+* You can now manage certificates for API Server and Ingress Controller by using the {cert-manager-operator}. For more information, see 
 
-[id="cert-manager-operator-1.11.1-bug-fixes"]
-=== Bug fixes
+* With this release, the scope of {cert-manager-operator}, which was previously limited to the AMD64 platform, has now been expanded to include support for managing certificates on IBM Z, IBM Power and ARM64 platforms.
 
-* Previously, the `cm-acme-http-solver` pod did not use the latest published Red Hat image `registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9`. With this release, the `cm-acme-http-solver` pod uses the latest published Red Hat image `registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9`. (link:https://issues.redhat.com/browse/OCPBUGS-10821[*OCPBUGS-10821*])
+* With this release, you can use DNS over HTTPS (DoH) for performing the self-checks during the ACME DNS01 verification. The DNS self-check method can be controlled by using the command line flags, `--dns01-recursive-nameservers-only` and `--dns01-recursive-nameservers`. You can use DoH when all the traffic must be HTTP or HTTPS traffic, for example, when using `HTTPS_PROXY`.
```

Comment:
```suggestion
* With this release, you can use DNS over HTTPS (DoH) for performing the self-checks during the ACME DNS-01 challenge verification. The DNS self-check method can be controlled by using the command line flags, `--dns01-recursive-nameservers-only` and `--dns01-recursive-nameservers`.
```
We can also use DoH without HTTP / HTTPS proxy setup on the cluster.

Reply:
ok. This was something mentioned on the CM release note.